src/main/java/Sayo.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Scanner;
+
+public class Sayo {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        Task[] items = new Task[100];
+        int itemsCount = 0;
+
+        System.out.println("Hello! I'm Sayo and it's great to see you! \n" + "What can I do for you today? \n");
+
+        String input = " ";
```

Comment:
Naming convention is not descriptive enough here, consider changing the name to userInput